src/hooks/protections/can_apply_data.cpp

```diff
@@ -773,9 +773,9 @@ namespace big
 				{
 					if (object && g_local_player && g_local_player->m_net_object)
 					{
-						if (is_in_vehicle(g_local_player, g_local_player->m_vehicle) && g_local_player->m_vehicle->m_net_object
-						    && g_local_player->m_vehicle->m_net_object->m_object_id == object->m_object_id)
-							return true; // vehicle kick?
+						// if (is_in_vehicle(g_local_player, g_local_player->m_vehicle) && g_local_player->m_vehicle->m_net_object
+						//     && g_local_player->m_vehicle->m_net_object->m_object_id == object->m_object_id)
+						// 	return true; // vehicle kick?
```

Comment:
We could've just removed it since we've got version control.

Though it might not take too long until this gets fixed so it can stay for a bit.

Reply:
I hesitated cuz it is not mine and it might be an easy fix.